dd-java-agent/instrumentation/vertx/src/test/groovy/VertxServerTest.groovy

```diff
@@ -67,8 +68,25 @@ class VertxServerTest extends AgentTestRunner {
           }
         }
         span(1) {
+          assert span(1).operationName.endsWith('.handle')
```

Comment:
Can this be not be hardcoded?  It's likely suffering from cardinality issues otherwise.